programs/axelar-solana-gateway/src/processor/initialize_message_payload.rs

```diff
@@ -128,6 +142,7 @@ impl Processor {
         let mut message_payload_account_data = message_payload_account.try_borrow_mut_data()?;
         let message_payload: MutMessagePayload<'_> = (*message_payload_account_data).try_into()?;
         *message_payload.bump = bump_seed;
+        *message_payload.payload_hash = incoming_message.payload_hash;
```

Comment:
I think we can drop this. This will imply the message has already been committed